docs/application/web/guides/data/file-system.md

```diff
@@ -108,149 +107,261 @@ To manage file storages:
+   var destination = 'images/copyOfExampleFile.jpg';
    function onSuccess() {
        console.log('success');
    }
+   var errorCallback = function(error) {
+       console.log(error);
+   }
+   tizen.filesystem.copyFile(filePathAndName, destination, true, onSuccess, errorCallback);
+   ```
+
+2. To copy a directory, use the `copyDirectory()` method. The following example copies the directory to the `documents/` directory. Since the third parameter is set to `true`, any existing file or directory with the same name in the target directory is overwritten.
+
+   ```
+   var directoryPathAndName = 'downloads/exampleDirectory';
+   var destination = 'documents/copyOfExampleDirectory';
+   function onSuccess() {
+       console.log('success');
+   }
+   var errorCallback = function(error) {
+       console.log(error);
+   }
+   tizen.filesystem.copyDirectory(directoryPathAndName, destination, true, onSuccess, errorCallback);
+   ```
+
+3. To move a file, use the `moveFile()` method. The following example moves the files to the `music/` directory. Since the third parameter is set to `false`, no existing files with the same name in the target directory are overwritten.
 
-   for (var i = 0; i < files.length; i++) {
-       documentsDir.copyTo(files[i].fullPath, 'images/backup/' + files[i].name,
-                           true, onSuccess);
+   ```
+   var filePathAndName = 'downloads/exampleFile.mp3';
+   var destination = 'music';
+   function onSuccess() {
+       console.log('success');
    }
+   var errorCallback = function(error) {
+       console.log(error);
+   }
+   tizen.filesystem.moveFile(filePathAndName, destination, true, onSuccess, errorCallback);
    ```
 
-4. To move a file or directory, use the `moveTo()` method. The following example moves the files to the `images/newFolder/` directory. Since the third parameter is set to `false`, no existing files with the same name in the target directory are overwritten.
+4. To move a directory, use the `moveDirectory()` method. The following example moves the directory to the `camera/` directory. Since the third parameter is set to `true`, existing file or directory with the same name in the target directory is overwritten.
 
    ```
-   var files; /* Assume that this is an array of File objects */
+   var directoryPathAndName = 'documents/exampleDirectory';
+   var destination = 'camera';
+   function onSuccess() {
+       console.log('success');
+   }
+   var errorCallback = function(error) {
+       console.log(error);
+   }
+   tizen.filesystem.moveDirectory(directoryPathAndName, destination, true, onSuccess, errorCallback);
+   ```
 
-   for (var i = 0; i < files.length; i++) {
-       documentsDir.moveTo(files[i].fullPath, 'images/newFolder/' + files[i].name,
-                           false, onSuccess);
+5. To rename either a file or a directory, use the `rename()` method. The following example changes file name to the `newName.txt`.
```

Comment:
to the `newName.txt`:

Use a ":" at the end of the sentence as we have used the word following.

Reply:
Done.